content/get-started/getting-started-with-git/caching-your-github-credentials-in-git.md

```diff
@@ -105,3 +105,25 @@ For more options for storing your credentials on Linux, see [Credential Storage]
 <br>
 
 For more information or to report issues with GCM, see the official GCM docs at "[Git Credential Manager](https://github.com/GitCredentialManager/git-credential-manager)."
+
+## Other Git credential helpers
```

Comment:
I cannot speak for this section here but I hesitate whether GitHub officially supports these other `git` credential helpers.  Again, the 2 main credential helps I've seen supported are the GitHub CLI and GCM.  I don't think GitHub needs to be an authority on all `git` credential helpers nor do I think we support everything.

Reply:
@andyfeller, it is certainly helpful for GitHub to define which helpers are *recommended*. I argue that it is important for this page to be clear about whether other credential helpers are *allowed*.

The fact that GitHub has a mechanism for detecting OAuth apps other than GitHub CLI and GCM, and docs on [Approving OAuth apps for your organization](https://docs.github.com/en/organizations/managing-oauth-access-to-your-organizations-data/approving-oauth-apps-for-your-organization), gives me the message that other credential helpers are allowed. If they are allowed, then I think the docs should say that somewhere, rather than be silent.